app/src/main/res/values/strings.xml

```diff
@@ -147,29 +147,29 @@
     <string name="moveBarcodeToCenterOfScreen">Center the barcode on the screen</string>
 
     <string name="noBarcodeFound">No barcode was found</string>
-    <string name="errorReadingImage">Error reading image</string>
+    <string name="errorReadingImage">Could not read the image</string>
 
     <string name="balance">Balance</string>
     <string name="currency">Currency</string>
     <string name="points">Points</string>
 
     <string name="parsingBalanceFailed"><xliff:g>%s</xliff:g> does not seem to be a valid balance.</string>
-    <string name="chooseImportType">Import data from which app?</string>
+    <string name="chooseImportType">Import data from?</string>
     <string name="app_loyalty_card_keychain">Loyalty Card Keychain</string>
 
     <string name="privacy_policy">Privacy Policy</string>
-    <string name="privacy_policy_popup_text">Many app stores require apps to show their privacy policy on first start. Here is ours:\n\nWe collect NO DATA AT ALL and our app is Open Source so anyone can confirm this to be true.</string>
+    <string name="privacy_policy_popup_text">Privacy policy notice (required by some app stores):\n\nWe collect NO DATA AT ALL, which anyone can confirm since our app is libre software.</string>
```

Comment:
I believe we had a discussion about this before: nobody knows what "libre software" is. "Open Source" is a much more well-known term.

Aside from that it's a definitely better, yeah. I wrote the original string in annoyance (and it may go away again completely soon anyway if Huawei doesn't answer my email about this subject soon: https://twitter.com/SylvieLorxu/status/1377208451227598849)

Reply:
We did, but you must have seen it from my point of view, seeing as it is
<string name="app_license">Copylefted libre software, licensed GPLv3+.</string>
(and this would now be consistent with that).
I think you closed my effort to shorten down and simplify the Fastlane stuff, where it might have gone the other way.
No discussion to be had there, if memory serves me right.

Reply:
Well, that string didn't originally say Open Source though, that originally just said "Licensed under the GPLv3".

Aside from that, to be quite frank, given the FSF decided to be as disrespectful to the Free Software community as to reinstate RMS into a position of power behind everyone's back (hell, even the people talking at the conference didn't know, so they couldn't decide themselves if they felt okay with it or not: https://twitter.com/fsf/status/1374399897558917128) I feel even less tempted to use any of the FSF's phrasings. While I honestly find ESR way more problematic than RMS, the OSI distanced themselves from him ages ago and he is nowhere to be seen on the staff list: https://opensource.org/volunteersandstaff.

Reply:
Curiously I am on the opposite side of the fence with RMS at the FSF. (Did read some of that Twitter stuff, and the one case I did research is presented erroneously.)
I think ESR tried to distance himself from the FSF/GNU crowd, and didn't realize the direction meant he would get canned down the road. I have yet to read his book, which is the part I would be interested in. I see him in the changelogs here and there.

I however feel strongly compelled not to use FSF language. "Copylefted libre software" manages to avoid both the OSI and the FSF.

Reply:
Yeah, the most known case is indeed presented erroneously (he never stated that abuse victim WAS willing, just that someone couldn't have known she wasn't given she was likely coerced to lie and ACT willing), which is very problematic because 1) reporting should be honest and 2) it makes the actual point much weaker. It is the main reason I didn't sign the letter, because I find it pretty disturbing that even though most claims are sensible an incorrect one gets added just like that. Nobody should have their words twisted and then used against them, it's simply not reasonable.

As I understand it, ESR's plan was to make "Free Software" less "scary" to companies by de-emphasizing the "freedom" aspect of the name. If you look at his blog or whatever, he has... quite "conservative" opinions regarding firearms and women and all. Not a fan myself.

I do believe libre was a term explicitly suggested by the FSF though. Maybe we should just go for "Free and Open Source (FOSS)"? The FOSS shortening makes it hopefully clear enough that the free doesn't mean gratis. Pfff, discussions on these term are always so complex... I used Open Source because it is simply the most well-known term, especially among non-technical users, but I definitely don't have a strong OSI preference.

Reply:
It would surprise me if me and ESR agreed on everything, even more after sharing our views. Bruce Perens also has some involvement in using "Open Source" as rebranding of both "free software" and what was a different meaning originally. However changed, reformed, self-acclaimed wrong or cancelled any of these characters get, I judge their other opinions in their own accord.
"Open Source" may be the most known as a brand, but it is the brand of intent I don't support. Precisely in this effort it succeeded in not carrying over to communicating the totality of libre freedoms.
Instead it means ~stuff that is gratis for some reason, and ~the source is in the open, anyone can read it

The FSF runs "Free software" with a consistency seemingly only made possible by RMS himself.
I have been using "libre" for a number of years now.

FOSS also manages to not stick, and is a compromise that is truly complex for no benefit.
FLOSS epitomizes this, and the way one thing qualifies another doesn't carry over to other languages.
Add to that how it sometimes "Free and Open Source software", and it becomes truly misleading.

To illustrate a point, "Open Source" means something completely different in translation, because it isn't a proper noun term, much less a trademarked one. Moreover, some software is "open source", like Vivaldi (the browser).
You are a technical user and use these two spellings between the app description and the app itself.
I think libre software gives people half a chance at understanding what something is, and why it is important.

Reply:
Oh, those translations are very interesting. I would have expected way different results. Combining that with LibreOffice being probably the most well-known libre piece of software... yes, that makes sense.